db/migrations/mysql/000052_create_public_channels.up.sql

```diff
@@ -63,18 +63,21 @@ SET @preparedStatement = (SELECT IF(
         AND index_name = 'idx_publicchannels_search_txt'
     ) > 0,
     'SELECT 1',
-    'CREATE FULLTEXT INDEX idx_publicchannels_search_txt ON PublicChannels(name, displayname, purpose);'
+    'CREATE FULLTEXT INDEX idx_publicchannels_search_txt ON PublicChannels(Name, DisplayName, Purpose);'
 ));
 
 PREPARE createIndexIfNotExists FROM @preparedStatement;
 EXECUTE createIndexIfNotExists;
 DEALLOCATE PREPARE createIndexIfNotExists;
 
 CREATE PROCEDURE MigratePC ()
-BEGIN
-	IF(NOT EXISTS (
-		SELECT
-			1 FROM PublicChannels)) THEN
+BEGIN DECLARE
```

Comment:
Is this a syntax error? How did it work before then?

Reply:
SRE team had trouble while running this manually, so I replaced with a one I and them could run. It was working on my local machine.